window.go

```diff
@@ -26,6 +26,14 @@ func (w *Window) AddEventListener(typ string, h EventHandler) {
 	}))
 }
 
+func (w *Window) Open(url, windowName, windowFeatures string) {
+	w.v.Call("open", url, windowName, windowFeatures)
```

Comment:
As a though, the `windowFeatures` can be a `map[string]string`, and the function can convert it to `k1=v1,k2=v2`.

What do you think?

Reply:
actually yeah that makes sense.

Reply:
but what about the string escaping?

Reply:
@dennwc I have added the same as per your suggestion, can you review it 😄

Reply:
not sure about string escaping, we will need to check it separately